drivers/rtc/CMakeLists.txt

```diff
@@ -3,3 +3,5 @@
 zephyr_library()
```

Comment:
In commit `drivers: rtc: add system call handlers`, I can understand that it is easy to just copy the commit subject to the body but we should try to avoid this. It is really annoying later if one looks the commits and tries to understand what the commit is about and both the subject and body are same (not really applicable for this commit as it is easy to understand what the commit does but as a general note). For example here the commit body could say `Add system call handlers so that the RTC get/set functions can be called from userspace`. So the commit body should tell why we are doing the change, it is usually not necessary to document what the commit does as that can be usually seen from the code.